src/infrastructure/KyRequester.ts

```diff
@@ -7,6 +7,17 @@ import { skipAllCaps } from './Utils';
 const methods = ['get', 'post', 'put', 'delete'];
 const KyRequester = {} as Requester;
 
+function responseHeadersAsObject(response) {
+    const headers = {}
+    const keyVals = [...response.headers.entries()]
```

Comment:
Object.fromEntries would be a nice polyfill:
https://medium.com/@selvaganesh93/javascript-whats-new-in-ecmascript-2019-es2019-es10-35210c6e7f4b